src/mqueryfront/src/QueryMonaco.js

```diff
@@ -74,6 +74,9 @@ class QueryMonaco extends Component {
         if (this.props.error !== prevProps.error) {
             this.setError(...this.props.error);
         }
+        if (prevProps.rawYara !== "" && this.props.rawYara === "") {
```

Comment:
what triggered this change? was something broken?

Reply:
It should be enough to set attribue `value=""` to `<Editor /> `component to clear its content. But it didn't work. The `value` attribute is set to `{this.props.rawYara}`. That's why I added code to clear editor content when corresponding prop.rawYara changes from non empty string to empty string.

Reply:
This code is unnecessary if the Monaco component is successfully initialized. I will get rid of this code.